### ISSUE TITLE
When opening service that binds 'host' I get a black page.

### ISSUE THREAD

**Issue by atmosx** [swarmpit/swarmpit#336]
**Description**

I'm using metricbeat with the following mounts:

```
        mnt = ['/var/run/docker.sock:/var/run/docker.sock:ro',
               '/sys/fs/cgroup:/hostfs/sys/fs/cgroup:ro',
               '/proc:/hostfs/proc:ro',
               '/:/hostfs:ro']
```

And networks

```
networks=['host']
```

On a swarm cluster. When I click on the `metricbeat` service page, I get a black page. In the page source code I see the following:

```

<script>
--
  | window.isUpdateAvailable = new Promise(function (resolve, reject) {
  | if ('serviceWorker' in navigator && ['localhost', '127'].indexOf(location.hostname) === -1) {
  | window.addEventListener('load', function () {
  | navigator.serviceWorker.register('/sw.js').then(function (registration) {
  | console.log("ServiceWorker registration successful with scope: ", registration.scope);
  | registration.onupdatefound = function () {
  | var installingWorker = registration.installing;
  | installingWorker.onstatechange = function () {
  | switch (installingWorker.state) {
  | case 'installed':
  | if (navigator.serviceWorker.controller) {
  | resolve(registration);
  | } else {
  | resolve(false);
  | }
  | break;
  | }
  | }
  | }
  | }).catch(function (err) {
  | console.log("ServiceWorker registration failed: ", err);
  | reject(err);
  | });
  | });
  | }
  | })
  | </script>


```

Comes from `main.js`.

**What happens:**

I get a blank service page.

**What should happen:**

I would like to see the a service page or an explanatory error message.

**Additional information (e.g. docker version, cluster setup,...):**

Swarmpit 1.6
Docker Server version 18.09.2

**Comment by nohaapav** [comment]
@atmosx Could you past your stack so i can try by myself ?

**Comment by nohaapav** [comment]
Works for me pretty well with following stack:

<img width="899" alt="Screen Shot 2019-03-24 at 1 00 20 PM" src="https://user-images.githubusercontent.com/4179104/54879037-dc221d80-4e34-11e9-9d6e-4aa6ed3eaf3b.png">

See:

<img width="1678" alt="Screen Shot 2019-03-24 at 12 59 59 PM" src="https://user-images.githubusercontent.com/4179104/54879040-e2b09500-4e34-11e9-8e75-06a7dff61189.png">

**Comment by atmosx** [comment]
Weird. I'm using a python dockerAPI to launch services. My definition is exactly as the one you have above but here's what I see: https://www.dropbox.com/s/nq518mkz1g65cn8/metricbeast-swarmpit.mp4?dl=0

**Comment by nohaapav** [comment]
Yes host network is the problem. Was able to reproduce this scenario finally. Could you paste your stack here ? I'm wondering about host network setup.

**Comment by nohaapav** [comment]
Problems seems to be with fetching of host network: 

When `http://0.0.0.0:3449/api/services/metricbeat_metricbeat/networks` is called invalid response is returned:

```
[{"labels":null,"ingress":null,"enableIPv6":null,"created":null,"scope":null,"internal":null,"id":null,"ipam":{"subnet":null,"gateway":null},"stack":null,"options":[],"networkName":null,"driver":null,"attachable":null,"serviceAliases":null}]
```

This is because we can't fetch given network from network list by id.

See:
<img width="444" alt="Screen Shot 2019-03-25 at 6 14 43 PM" src="https://user-images.githubusercontent.com/4179104/54940138-6c8e5a00-4f2a-11e9-8f02-2f2f3acca896.png">

There is no such network with given id in list:
<img width="513" alt="Screen Shot 2019-03-25 at 6 14 57 PM" src="https://user-images.githubusercontent.com/4179104/54940163-7748ef00-4f2a-11e9-9673-f685c68b8b11.png">

**Comment by atmosx** [comment]
Hello, the `host` network, uses the `host` driver and has `local` scope.

**Comment by nohaapav** [comment]
Wondering why network target from service definition does not match the host network from network list. Strange is that if i inspect the resource with docker inspect {id} i can see host network but this one has swarm scope .. and it's not even listed ..

`[
    {
        "Name": "host",
        "Id": "vu8gf4cw2sfpdybmfg4ar53b1",
        "Created": "2017-09-17T12:23:38.709837442Z",
        "Scope": "swarm",
        "Driver": "host",
        "EnableIPv6": false,
        "IPAM": {
            "Driver": "",
            "Options": null,
            "Config": []
        },
        "Internal": false,
        "Attachable": false,
        "Ingress": false,
        "ConfigFrom": {
            "Network": ""
        },
        "ConfigOnly": false,
        "Containers": null,
        "Options": null,
        "Labels": {
            "com.docker.swarm.predefined": "true"
        }
    }
]`

**Comment by nohaapav** [comment]
And even if we fix this BLANK SCREEN behaviour host network will be listed only if on same node as swarmpit_app because it has local scope. We need to fetch local resources such networks/volumes via agent which is not currently supported.

**Comment by lumir-mrkva** [comment]
will be fixed by #444
